styles/styles.css

```diff
@@ -0,0 +1,389 @@
+* {box-sizing: border-box;}
+
+* {font-family: Georgia, serif;}
+
+
+header {
+  display: flex;
+
+  flex-direction: column;
+  padding: 2em;
+
+  background: rgb(115,136,153);
+  background: linear-gradient(0deg, rgba(115,136,153,1) 0%, rgba(9,60,102,1) 100%);
+  background-size: auto;
+  background-image: url('../images/header-image.jpg');
+
+
+
+}
+
+
+.logo {
+  width: 70%;
+}
+
+.logo img {
+  width: 60%;
+}
+
+.logo h1 {
+  font-size: 2em;
+}
+
+header button {
+  display: flex;
+  width: 10em;
+  justify-content: center;
+  padding: .5em;
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+}
+
+.checkout {
+  justify-content: center;
+  flex-direction: column;
+  align-self: flex-end;
+  display: flex;
+}
+
+i {
+  color: white;
+  font-size: 1em;
+  display: flex;
+  align-self: flex-end;
+  margin-bottom: 1em;
+}
+
+body {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+nav {
+  background-color: #e1e7eb;
+  width: 100%;
+  padding: 1.5em;
+}
+
+nav li {
+display: inline;
```

Comment:
If you use inline block you might gain some more control with styling. Just a heads up for future projects. You can adjust things like padding and margin on inline-block elements